kqueen_ui/asset/dynamic/scss/main.scss

```diff
@@ -497,6 +497,20 @@ button.add-field {
   }
 }
 
+i.mdi-help-circle {
+  font-size: 22px;
+  margin-left: 4px;
+  position: relative;
+  top: 2px;
+  color: $mira-orange;
```

Comment:
Maybe purple is better as it's not so distractive and it doesn't look like a notification